### PR TITLE
Added launch all modules function.

### PR DESCRIPTION
**Pull request by bwkiefer**
Adds launch all function to api to launch existing modules.

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| Doc PR        | https://github.com/pm2-hive/pm2-hive.github.io/pulls

*Please update this template with something that matches your PR*

### CODE REVIEW

**Comment by bwkiefer** [comment]
@saevon

File: lib/API.js [hunk 1]
```diff
@@ -203,6 +203,17 @@ API.prototype.disconnect = API.prototype.close = function(cb) {
 };
 
 /**
+ * Launch all modules
+ *
+ * @param {Function} cb callback once pm2 is ready for commands
+ */
+API.prototype.launchAll = function(cb) {
```
Comment (Saevon) [inline lib/API.js]: rename to `launchModules`? or make sure it launches everything (and thus test it?)

File: test/programmatic/api.mocha.js [hunk 2]
```diff
@@ -269,4 +269,37 @@ describe('API checks', function() {
     });
   });
 
+  describe('Should start up modules', function() {
+    var Modularizer = require('../../lib/API/Modules/Modularizer');
+
+    after(function(done) {
```
Comment (Saevon) [inline test/programmatic/api.mocha.js]: add more tests such as:
1) no modules installed
2) multiple modules installed
3) launching when the modules are already launched?

P.S. maybe not.. considering this just calls an internal method...

File: lib/API.js [hunk 3]
```diff
@@ -203,6 +203,17 @@ API.prototype.disconnect = API.prototype.close = function(cb) {
...
+  Modularizer.launchAll(this, function(err_mod) {
+    return cb(err_mod);
```
Comment (Saevon) [inline lib/API.js]: why not pass the cb directly

File: test/programmatic/api.mocha.js [hunk 4]
```diff
@@ -269,4 +269,37 @@ describe('API checks', function() {
...
+      PM2.destroy(done);
+    });
+
+    it('should start up modules', function(done) {
+      this.timeout(1000);
```
Comment (Saevon) [inline test/programmatic/api.mocha.js]: why a timeout?

**Pull request closed without merging**